mailexporter.go

```diff
@@ -311,11 +311,14 @@ func probe(c smtpServerConfig, p payload) {
 
 // monitor probes every MonitoringInterval if mail still gets through.
 func monitor(c smtpServerConfig) {
+	// keep a random timedelta between monitoring jobs to reduce interference
```

Comment:
I think "delay start of monitoring randomly to desync the probing of the monitoring-coroutines" would be better suited as a comment, because with this implementation it is no longer a timedelta *between* monitoring jobs.